### PR TITLE
feat: Enhance submission metadata

### PR DESCRIPTION
**Pull request by ktagowski**

### CODE REVIEW

**Review by asawczyn — DISMISSED** [review]

File: embeddings/evaluator/leaderboard.py [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    ner = "Named Entity Recognition"
+    paraphrase_classification = "Paraphrase Classification"
+    political_advertising_detection = "Political Advertising Detection"
+    pos = "Part-of-speech Tagging"
+    punctuation_restoration = "Punctuation Restoration"
+    qa_classification = "Q&A Classification"
+    sentiment_analysis = "Sentiment Analysis"
+
+
+class LeaderboardDomain(str, Enum):
```
Comment (laugustyniak) [inline embeddings/evaluator/leaderboard.py:35]: Can we change it and assign domain such as 

1. News - News articles from a variety of sources and topics, such as politics, sports, business, and entertainment.
2. Social Media - Social media posts from various platforms, such as Twitter, Facebook, and Instagram, to capture informal language and user-generated content.
3. Conversational - Conversational text, such as chat logs or customer service transcripts, to capture natural language interactions.
4. Scientific and technical - Scientific and technical documents from various fields, such as medicine, engineering, and law, to capture domain-specific language and terminology.
5. Literary - Literary texts, such as books, poems, and plays, to capture creative writing styles and nuances.
6. Legal - Legal texts can include various types of documents such as contracts, statutes, regulations, court opinions, and legal briefs. They may also contain specific features such as citations, references to legal cases, and specialized terminology.
7. Other - if anything does not fit.

Wikipedia looks more like source not a genre.

Comment (ktagowski) [reply embeddings/evaluator/leaderboard.py:35]: It is based on the LEPISZCZE paper taxonomy https://arxiv.org/pdf/2211.13112.pdf (Table 1)

Comment (laugustyniak) [reply embeddings/evaluator/leaderboard.py:35]: I know, and I am not sure it is a good taxonomy. :D

Comment (laugustyniak) [reply embeddings/evaluator/leaderboard.py:35]: I do not have any taxonomy in my mind right now. We can start with the example described above.

**Review by laugustyniak — APPROVED** [review]

**Pull request merged**